chainerx/__init__.pyi

```diff
@@ -322,6 +322,10 @@ class ndarray:
 
     def __neg__(self) -> ndarray: ...
 
+    def __pow__(self, arg0: tp:Any) -> ndarray: ...
+
+    def __rpow__(self, arg0: tp.Any) -> ndarray: ...
```

Comment:
How about sorting in lexicographical order of attribute names for consistency?

Reply:
Thanks for the comment, I didn't notice that. Should be fixed now.